src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -85,6 +86,15 @@ export class TaskListBuilder {
         )
       )
     }
+    if (draft.isResponseFullyAdmitted() && draft.isResponseFullyAdmittedWithPayBySetDate()) {
```

Comment:
That is wrong. Include party type checking.

Reply:
Included a check for INDIVIDUAL and SOLE_TRADER_OR_SELF_EMPLOYED party type